### ISSUE TITLE
Dashboard properties are not filtered

### ISSUE THREAD

**Issue by jharkins2** [tsgrp/HPI#1731]
When viewing properties in a dashboard, the properties are not filtered if there is an applied filter:
![dashboard_filter](https://user-images.githubusercontent.com/22036857/42603496-b5724612-8533-11e8-8746-aaaa00b08efc.gif)

**Comment by gsteimer** [comment]
@jharkins2 - are you only seeing this for the Recent Objects dashlet?  Does it work in Saved Search dashlets?

**Comment by jharkins2** [comment]
Yep sorry about that, we're only seeing that with recent objects, not saved search dashlets:
![capture](https://user-images.githubusercontent.com/22036857/42638416-312a8efc-85b3-11e8-84df-04e9b6282291.PNG)

**Comment by gsteimer** [comment]
Huh - looking at this again, it looks like the date is properly filtered, but not the creator name.  The date is filtered wrong too.  In the OTC, it was configured as a date time, but the dashboard is displaying just the date.

**Comment by gsteimer** [comment]
Moved to Jira: https://alfresco.atlassian.net/browse/SEP-56
